src/test/java/kitchenpos/application/MenuGroupServiceTest.java

```diff
@@ -0,0 +1,69 @@
+package kitchenpos.application;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import kitchenpos.domain.MenuGroup;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.transaction.annotation.Transactional;
+
+
+@SpringBootTest
+@Transactional
+class MenuGroupServiceTest {
+
+    @Autowired
+    private MenuGroupService menuGroupService;
+
+
+    @Test
+    @DisplayName("menu group 생성")
+    void menu_create_test() {
+        //given
+        MenuGroup menuGroupRequest = MENU_GROUP_REQUEST_생성("양식");
+        //when
+        MenuGroup createdMenuGroup = MENU_GROUP_생성_테스트(menuGroupRequest);
+
+        //then
+        Assertions.assertAll(() -> {
+            assertThat(createdMenuGroup.getId()).isNotNull();
+            assertThat(createdMenuGroup.getName()).isEqualTo(menuGroupRequest.getName());
+        });
+    }
+
+    @Test
+    @DisplayName("menu group 리스트 조회")
+    void menu_show_test() {
+        //given
+        MenuGroup menuGroup1 = MENU_GROUP_생성_테스트(MENU_GROUP_REQUEST_생성("양식"));
+        MenuGroup menuGroup2 = MENU_GROUP_생성_테스트(MENU_GROUP_REQUEST_생성("중식"));
+
+        //when
+        List<MenuGroup> list = menuGroupService.list();
+
+        //then
+        Assertions.assertAll(() -> {
+            List<String> collect = list.stream().map(MenuGroup::getName).collect(Collectors.toList());
+            List<String> menuGroups = Arrays.asList(menuGroup1.getName(), menuGroup2.getName());
+            assertThat(collect).containsAll(menuGroups);
```

Comment:
assertj 검증문을 활용하면 아래와 같이 작성할 수 있습니다.

```suggestion
            Assertions.assertThat(list)
                .extracting(MenuGroup::getName)
                .containsExactly(menuGroup1.getName(), menuGroup2.getName());
```